### DISCUSSION TITLE
1.18 versions crash when loading terrain

### DISCUSSION THREAD

**Discussion by viant22** [FabricMC/fabric-api#2648] [Player support]
My fabric crashes when I try lo load into a world. It freezes on loading terrain. I don't have this problem on the 1.19 version. i get error code -805306369 which basically says it doesn't have enough memory, but I gave it up to 16G of memory so that shouldn't be a problem. Especially because I can run the newer versions with 2G memory. Any help is appreciated. Thanks

**Comment by warjort** [comment]
That error code is CFFFFFFF or Application Hung
So, it looks like the OS ended the process because it was not responding. 
You might have a file beginning `hs_err_pid` in your minecraft folder?
If you do, this should show what each thread was doing when it crashed.

WARNING: Before you share the hs_err_pid file, make sure you delete any personal information such as your minecraft login access token. That crash report does not obfuscate such things.
